mesos-docker/run/run.sh

```diff
@@ -524,59 +544,59 @@ function show_help {
 
   Options:
 
+  --cpu-th the percentage of the host memory to use for slaves. Default: 0.5.
+  --hadoop-binary-file the hadoop binary file to use in docker configuration (optional, if not present tries to download the image).
   -h|--help prints this message.
-  -q|--quiet no output is shown to the console regarding execution status.
-  --number-of-slaves number of slave mesos containers to create (optional, defaults to 1).
-  --hadoop-binary-file  the hadoop binary file to use in docker configuration (optional, if not present tries to download the image).
-  --spark-binary-file  the hadoop binary file to use in docker configuration (optional, if not present tries to download the image).
   --image-version  the image version to use for the containers (optional, defaults to the latest hardcoded value).
-  --with-zk sets master as a zookeeper node.
-  --with-mararthon starts marathon node requires zookeeper.
+  --mem-th the percentage of the host cpus to use for slaves. Default: 0.5.
+  --mesos-master-config parameters passed to the mesos master (specific only and common with slave).
```

Comment:
at mesos doc you have config params shared by  master and slave but also you have some specific to each of these.